modules/cnf-topology-aware-lifecycle-manager-apply-policies.adoc

```diff
@@ -370,5 +370,5 @@ $ oc get csv -n <operator_namespace>
 [source,terminal]
 ----
 NAME                    DISPLAY                     VERSION   REPLACES   PHASE
-cluster-logging.5.4.2   Red Hat OpenShift Logging   5.4.2                Succeeded
+cluster-logging.v6.2.1  Red Hat OpenShift Logging   6.2.1                Succeeded
```

Comment:
Should the `v` be included before `6.2.1` on line 373?

Reply:
Yes, v is part on the Operators name, so the proposed change is accurate.

```
[kni@registry ~]$ oc get csv -n openshift-logging
NAME                     DISPLAY                     VERSION   REPLACES   PHASE
cluster-logging.v6.2.1   Red Hat OpenShift Logging   6.2.1                Succeeded
```